andela/learning-map-front
# #149954569 Implement Add Comment & Load Comments functionality

#### What does this PR do?
- Loads comments belonging to a contribution as soon as the contribution modal is active
- Gives users the ability to post comments on a given contribution

#### Description of Task to be completed?
A user of learning map should be able to share his/her opinion of a contribution on `Learning Map`. The use of the comment component ensures a logged in user is able to comment on a contribution.
On viewing the contribution too, he/she would see the latest comments for a contribution.

#### How should this be manually tested?
After cloning the repository:
- Run yarn install
- Run yarn start
- Access the landing page via localhost:3000 and click on the `LOGIN TO CONTRIBUTE` button.
- Log in with your `andela` e-mail address
- You'll be directed to the contribution page
- Click on any of the contribution and comment on it
- You'll see your comment appear in the comment box.

#### What are the relevant pivotal tracker stories?
[#149954569](https://www.pivotaltracker.com/story/show/149954569)

#### Screenshots (if appropriate)
<img width="1669" alt="screen shot 2017-08-25 at 8 55 39 am" src="https://user-images.githubusercontent.com/25608335/29704816-453b8816-8973-11e7-9abf-30cae23ce249.png">

![screen shot 2017-08-25 at 6 38 08 pm](https://user-images.githubusercontent.com/25608335/29725607-9fddb310-89c4-11e7-909f-aee4adbbad57.png)


#### Questions:
N/A
